mlrun/projects/project.py

```diff
@@ -3060,6 +3060,7 @@ def run(
         source: str = None,
         cleanup_ttl: int = None,
         notifications: list[mlrun.model.Notification] = None,
+        workflow_runner_node_selector: typing.Optional[dict[str, str]] = None,
```

Comment:
This should be part of `project.set_workflow` and not here. We need to be consistent with the way we work, see how we implemented setting the workflow runner image.

Reply:
discussed offline